webapp/src/components/modals/channel_subscriptions/channel_subscription_filter.tsx

```diff
@@ -269,7 +270,16 @@ export default class ChannelSubscriptionFilter extends React.PureComponent<Props
         };
 
         let valueSelector;
-        if (isEpicLinkField(this.props.field)) {
+        if (isCommentVisibilityField(field)) {
+            valueSelector = (
+                <JiraCommentVisibilitySelector
+                    {...selectProps}
+                    fieldName={field.name}
+                    value={value.values}
+                    onChange={this.handleEpicLinkChange}
```

Comment:
This should be a different function than `handleEpicLinkChange` right? Looks like that function isn't actually specific to epic links though, and does what we want here too, and it's being used in other places. Maybe we can change it to something like `handleValueChangeWithoutName`